src/test/java/org/folio/service/mapping/MappingServiceUnitTest.java

```diff
@@ -15,21 +22,41 @@
 @RunWith(MockitoJUnitRunner.class)
 public class MappingServiceUnitTest {
 
+  @Mock
+  private MappingSettingsProvider mappingSettingsProvider;
+  private String jobExecutionId = "67429e0e-601a-423b-9a29-dec4a30c8534";
+  private OkapiConnectionParams params = new OkapiConnectionParams();
+
+  @Before
+  public void mockSettings() {
+    Settings settings = new Settings();
+    JsonObject natureOfContentTerm_audioBook = new JsonObject()
+      .put("id", "96879b60-098b-453b-bf9a-c47866f1ab2a")
+      .put("name", "audiobook")
+      .put("source", "folio");
+    JsonObject natureOfContentTerm_autobiography = new JsonObject()
+      .put("id", "04a6a8d2-f902-4774-b15f-d8bd885dc804")
+      .put("name", "autobiography")
+      .put("source", "folio");
+    settings.addNatureOfContentTerms(Arrays.asList(natureOfContentTerm_audioBook, natureOfContentTerm_autobiography));
+    Mockito.when(mappingSettingsProvider.getSettings(jobExecutionId, params)).thenReturn(settings);
```

Comment:
My only concern with this that there will not be an end-end test if we are mocking this part. Can we just setup vertx mocks, instead of mocking the call. I think it is already done above in testing the client class, would be better to just use them here

Reply:
This is just a unit test, it is not related to end-to-end testing

Reply:
The idea of unit test is to cover only one piece of code - MappingService, no need to cover remote calls. We have tests for http clients to cover remote calls with vertx mocks

Reply:
Also, I plan to create a user story to rewrite EndToEnd test to use Vertx mocks

Reply:
Exactly, that's my point, we should not be adding separate stories to cover tests. It is a good idea to cover all the tests here

> The idea of unit test is to cover only one piece of code - 
I know what a unit test is Igor :) . My point is that we should be writing end-end-tests here, it's not much of an effort because we already have the code for it in the class that I mentioned